types/draft-convert/index.d.ts

```diff
@@ -1,73 +1,94 @@
-// Type definitions for draft-convert v2.1.5
+// Type definitions for draft-convert v2.1.10
 // Project: https://github.com/HubSpot/draft-convert
 // Definitions by: Agustin Valeriani <https://github.com/avaleriani/>
 //                 Munif Tanjim <https://github.com/MunifTanjim>
+//                 Vadim Chistokhvalov <https://github.com/vadim-ch>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.9
+// TypeScript Version: 3.9
 
 // Based on: https://github.com/HubSpot/draft-convert/issues/107#issuecomment-488581709 by <https://github.com/sbusch>
-declare module 'draft-convert' {
-  import {
-    ContentBlock,
-    ContentState,
-    DraftBlockType,
-    DraftInlineStyleType,
-    Entity,
-    RawDraftEntity
-  } from 'draft-js';
-  import { ReactNode } from 'react';
 
-  interface IConvertToHTMLConfig {
-    // Inline styles:
-    styleToHTML?: <T extends DraftInlineStyleType>(style: T) => Tag | void;
+declare module "draft-convert" {
+    import {
+        DraftEntityMutability,
+        RawDraftContentBlock,
+        ContentState,
+        DraftBlockType,
+        DraftInlineStyleType,
+        Entity,
+        RawDraftEntity,
+    } from "draft-js";
+    import { ReactNode } from "react";
 
-    // Block styles:
-    blockToHTML?: (block: ContentBlock) => Tag;
+    type RawDraftContentBlockWithCustomType<T> = Omit<RawDraftContentBlock, "type"> & {
+        type: T;
+    };
 
-    // Entity styling:
-    entityToHTML?: (entity: RawDraftEntity, originalText: string) => Tag;
-  }
+    type ExtendedHTMLElement<T> = (HTMLElement | HTMLLinkElement) & T;
 
-  type EntityKey = string;
+    interface IConvertToHTMLConfig<S, B, E> {
```

Comment:
Given this is a minor version bump, can you apply defaults for these type arguments so as not to break anyone who may have imported `IConvertToHTMLConfig` and used it without type arguments?